src/Host.h

```diff
@@ -125,7 +125,7 @@ class Host : public QObject
     void               unregisterEventHandler(const QString&, TScript * );
     void               raiseEvent( const TEvent & event );
     void               resetProfile();
-    std::tuple<bool, QString, QString, QString, QString> saveProfile();
+    std::tuple<bool, QString, QString, QString> saveProfile(const QString &saveLocation, bool syncModules = false);
```

Comment:
For a _Qt only_ data type I guess you might wish to consider a `QPair<bool, QVector<QString>>` :grinning: